lib/bot.js

```diff
@@ -42,8 +45,13 @@ class Bot {
     this._gasPreserver = new GasPreserver()
   }
 
-  // --------------------------------------------------------------------------
-
+  /**
+   * Initializes a bot by logging in with the given credentials. This must be run before your bot can be used.
+   *
+   * @param {InitOptions} options – An object expecting `username` and `paperkey` (which are pretty self-explanatory), as well as `verbose`, which says whether the bot should log much of what it's doing.
+   * @returns {Promise<?string>}
+   * @memberof Bot
+   */
```

Comment:
Quick thought on documentation @nathunsmitty.

We're already flow typing. Perhaps keeping the JSDOC syntax, but switching to a build system like [documentation.js](https://github.com/documentationjs/documentation) might be helpful as it supports reading flow types and can get `@param` and `@return` correctly.

I'll make a issue to research how well they jive together.